plugin.js

```diff
@@ -1,47 +1,51 @@
 'use strict'
 
-const semver = require('semver')
-const console = require('console')
 const extractPluginName = require('./stackParser')
-const { join, dirname } = require('path')
 
 let count = 0
 
 function plugin (fn, options = {}) {
   let autoName = false
 
-  if (typeof fn.default !== 'undefined') { // Support for 'export default' behaviour in transpiled ECMAScript module
+  if (typeof fn.default !== 'undefined') {
+    // Support for 'export default' behaviour in transpiled ECMAScript module
     fn = fn.default
   }
 
   if (typeof fn !== 'function') {
-    throw new TypeError(`fastify-plugin expects a function, instead got a '${typeof fn}'`)
+    throw new TypeError(
```

Comment:
Do you use different formatting rules?

Reply:
Standard was saying there was errors, just applied `standard --fix`

Reply:
Hmm, fair enough XD